x-pack/plugins/apm/server/lib/service_nodes/index.ts

```diff
@@ -83,6 +93,12 @@ const getServiceNodes = async ({
       name: bucket.key as string,
       cpu: bucket.cpu.value,
       heapMemory: bucket.heapMemory.value,
+      hostName:
```

Comment:
Since you are changing an API, I'd expect an API test to fail. Would be great if you can add one if it doesn't already exist.

Reply:
I already fixed the snapshot that was failing